wandb/util.py

```diff
@@ -810,6 +812,19 @@ def default(self, obj: Any) -> Any:
         return json.JSONEncoder.default(self, obj)
 
 
+class WandBJSONEncoderWeave(json.JSONEncoder):
```

Comment:
not strictly against, but is there a specific reason we add additional handler instead of using one of the others?

Reply:
The other handlers make `np.arrays` that are longer than 32 elements into histograms.  We want want to keep all arrays exactly as they are.